CONTRIBUTING.md

```diff
@@ -38,14 +38,16 @@ If you’re getting ready to report a bug in NUnit, figuring out how to create a
 
 ## Documentation
 
-Great documentation is essential for any open source project and NUnit is no exception. [Our documentation](https://docs.nunit.org/articles/nunit/intro.html) often lags behind the features that have been implemented or would benefit from better examples.
+Great documentation is essential for any open source project and NUnit is no exception. 
 
-A great place to start is by going through the [NUnit release notes](https://github.com/nunit/docs/wiki/Release-Notes) and ensuring that the documentation for new features is up to date.
+[Our documentation](https://docs.nunit.org/articles/nunit/intro.html) often lags behind the features that have been implemented or would benefit from better examples.
```

Comment:
I think this should be an xref to the article id. See other examples throughout docs, or I can fix it up later today before merge

Reply:
But will xref work from the ordinary git repo to the docfx docs ?